### DISCUSSION TITLE
原神的社区真的受不了

### DISCUSSION THREAD

**Discussion by JadenDamitao** [share121/inter-knot#2680] [其他]
现在的社区环境简直了,很多在说这不好那不好的,什么从坚定的米孝子转成坚定的米黑,先且不说站队和非黑即白的问题,很多主观评价的东西就算你玩了多长时间也不能代表全体玩家吧?如果说觉得原神项目组傲慢不听从玩家意见,那你干脆别玩了,树脂160觉得少,升到200了又说为什么没有隔壁崩铁的可溢出树脂

- 更新日志很多时候是不看的,
- 看了也觉得是没用的,
- 福利是永远不满足的,
- 一个米游老玩家玩家是能代表全体玩家的
- 说一句米哈游的好话是绝对扣上米孝子帽子的,
- 理智是不会有的,语言是永远尖锐的,视频是挂着原神创作激励的,
- 看到米游赚钱是眼红的,
- 自己充的钱是必须让米哈游原路退还的,
- 米哈游的员工是给冲钱就得跟奴隶一样优化工作的,
- 米哈游是不用赚钱的,
- 是不用为以后发展铺路的,
- 是必须卷得过别的厂商的,
- 评价是必须踩一捧一的

**现在原神最劝退我的是这个B社区**
不想说什么了,原神做好自己的事就好
剧情没觉得有什么问题
就跟看书一样
一个作品总会有争议的点,如果剧情你觉得在跟你喂史
那巨人所表达的思想是否更加危险,难道你要去冲巨人的作者说这个结局跟史一样
角色是有自己的故事的,就像人一样,总有人不喜欢自己身上的某一点
这是米哈游为了角色所写的故事,总会有争议,总会有不好
正是这样才像一个活着的"人"
如果说想让所有人满意
**干脆出门右转找一个AI陪聊**
还有幼稚的问题,除非改16+,公司搬到国外不被审查,这样熊辉有人觉得幼稚
个人感觉不算幼稚,应该算原神剧情的风格就这样,剧情感觉在这档子水平挺好的了

所以说玩原神千万不要看社区
只要自己玩就好
真正享受理解的玩家是基本不会发话的

**Comment by 7346454** [comment]
同感，虽然我不玩原神，但我看二创，他们在二创下带节奏，属实有点恶心人了<img src="https://bbs-static.miyoushe.com/static/2024/06/12/129937301e8578a6d2ee36d053828028_6462963966446141885.png" alt="克洛琳德-平静" width="75" >

**Comment by YiFang172** [comment]
你说得对，但是三个纠缠之缘，散兵喂shit

**Reply by cunjimu** [reply]
你说得对，实际上是13个纠缠之缘

**Comment by AAAzhaoxiangji** [comment]
同意，现在的氵社真的……<img src="https://bbs-static.miyoushe.com/static/2024/07/31/ed2b65b6899fc16528b1d10adfcb9a38_8705924363701631267.png" alt="云璃 白眼" width="75" >

**Comment by cunjimu** [comment]
都无所谓了，现在我就想把节奏控制在米游社之外

**Comment by lixiao189** [comment]
不管了，反正我b站直接屏蔽某个当皇帝的开放世界游戏的激励计划，首页直接清爽了。既然无法阻挡深渊我也只能构建我的虚假之天了。搞不懂有些极端玩家在干啥，你这样搞我不玩原神也不会去看你家游戏啊

**Comment by dalizi2333333** [comment]
你别说，起码这次5.0更新后，还在带节奏的真就有点问题了
都说4.8更新让人看清mxz和老玩家，那刚好，5.0更新就是看清是节奏狗还是老玩家的
让我看看躲起来的人都在哪呢（）
